planning/behavior_velocity_intersection_module/src/util.cpp

```diff
@@ -468,7 +478,9 @@ IntersectionLanelets getObjectiveLanelets(
         }
         detection_lanelets.push_back(conflicting_lanelet);
         if (!adjacent_followings.empty()) {
-          detection_lanelets.push_back(adjacent_followings.front());
+          for (const auto & adjacent_following : adjacent_followings) {
+            detection_lanelets.push_back(adjacent_following);
+          }
```

Comment:
@beyzanurkaya Does this snippet need to be inside the for-loop `for (const auto & conflicting_lanelet : conflicting_lanelets)` ? If this snippet is not outside of it, `detection_lanelets` variable will contain duplicate elements in `adjacent_followings` by the number of `conflicting_lanelets().size`.